components/Explorer/Explorer.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 import React from 'react';
```

Comment:
careful about using eslint-disable. Try to only use it for things that are still being worked on, but even then I would advise against (except in some edge cases where one is doing a bad pattern and needs to do refactor work before it can pass linting).